apis/gear/src/Keyring.ts

```diff
@@ -7,6 +7,8 @@ import { waitReady } from '@polkadot/wasm-crypto';
 
 import { decodeAddress } from './utils';
 
+const VARA_SS58_FORMAT = 137;
```

Comment:
Let’s export it? Would be useful because we’re specifying this constant in plenty of places.